bus/routes.go

```diff
@@ -1346,25 +1345,25 @@ func (b *Bus) settingsGougingHandlerPUT(jc jape.Context) {
 func (b *Bus) settingsPinnedHandlerGET(jc jape.Context) {
 	ps, err := b.ss.PinnedSettings(jc.Request.Context())
 	if errors.Is(err, sql.ErrSettingNotFound) {
-		b.logger.Warn("pinned settings not found, returning defaults")
-		jc.Encode(api.DefaultPinnedSettings)
+		ps = api.DefaultPinnedSettings
+	} else if jc.Check("failed to get pinned settings", err) != nil {
 		return
-	} else if jc.Check("failed to get pinned settings", err) == nil {
-		// populate the Autopilots map with the current autopilots
-		aps, err := b.as.Autopilots(jc.Request.Context())
-		if jc.Check("failed to fetch autopilots", err) != nil {
-			return
-		}
-		if ps.Autopilots == nil {
-			ps.Autopilots = make(map[string]api.AutopilotPins)
-		}
-		for _, ap := range aps {
-			if _, exists := ps.Autopilots[ap.ID]; !exists {
-				ps.Autopilots[ap.ID] = api.AutopilotPins{}
-			}
+	}
+
+	// populate the Autopilots map with the current autopilots
+	aps, err := b.as.Autopilots(jc.Request.Context())
+	if jc.Check("failed to fetch autopilots", err) != nil {
+		return
+	}
+	if ps.Autopilots == nil {
+		ps.Autopilots = make(map[string]api.AutopilotPins)
+	}
+	for _, ap := range aps {
+		if _, exists := ps.Autopilots[ap.ID]; !exists {
```

Comment:
Should add the testing back as well. Afaik there was a line in `TestNewTestCluster` that fetched the pin settings and made sure there was 1 autopilot with default pin values.

Reply:
It's actually kind of sneaky the way this happened. The test is still there but in `newTestCluster` we set `pinned` settings to the default. There's no real reason to do it because the test settings are the same as the default settings. I'm going to remove that because that would've caught this regression.